src/azure-cli/azure/cli/command_modules/appservice/utils.py

```diff
@@ -75,3 +75,11 @@ def call(*args, **kwargs):
                 time.sleep(interval_sec)
         return call
     return decorate
+
+
+def raise_missing_token_suggestion():
+    pat_documentation = "https://help.github.com/en/articles/creating-a-personal-access-token-for-the-command-line"
+    raise CLIError("GitHub access token is required to authenticate to your repositories. "
```

Comment:
recommend to use the exception type in azclierror, e.g.
https://github.com/Azure/azure-cli/blob/3eea7fd773722b7a3c40b8115f29a3bad39b70bc/src/azure-cli-core/azure/cli/core/azclierror.py#L142-L144